src/python_testing/TC_DA_1_7.py

```diff
@@ -50,6 +52,8 @@ def load_all_paa(paa_path: Path) -> dict:
                 if extension.oid == SubjectKeyIdentifier.oid:
                     skid = extension.value.key_identifier
                     paa_by_skid[skid] = (Path(filename).name, paa_cert)
+            except (ValueError, IOError) as e:
+                logging.error(f"Failed to load {filename}: {str(e)}")
```

Comment:
I would prefer to have a check against a "known bad certificate list" here and only allow those to pass with log-only. this means we still validate most of the certificates (almost all probably) and are aware which ones seem bad.

If we start failing, the known bad list can be expanded.

Reply:
Fixed